Python/bltinmodule.c

```diff
@@ -1584,34 +1591,33 @@ builtin_hex(PyObject *module, PyObject *number)
 }
 
 
-/* AC: cannot convert yet, as needs PEP 457 group support in inspect */
+/*[clinic input]
+iter as builtin_iter
+
+    v: object
```

Comment:
Not a fan of `v` but it's hard to come up with something better because of the two overloads.

Reply:
In the docs it is 

```
.. function:: iter(object)
              iter(object, sentinel)
```